### ISSUE TITLE
Support for Conjure from Jumpstart Historic Horizons

### ISSUE THREAD

**Issue by mainman879** [Cockatrice/Cockatrice#4410]
While most of Jumpstart Historic Horizons is difficult to impossible to implement in Cockatrice, one thing could be implemented in a fairly straightforward way (in the xml at least, I have no clue how difficult this would be to implement in the codebase). Conjure is a new ability in the set that creates a token card into zones like the graveyard or hand, and these token cards can exist in these zones forever.

1. We need a way to create a token that has "Destroy token when it leaves the table" disabled. I would suggest a new tag for cards like "movable".
2. There also needs to be a way to create a token into another zone, perhaps a tag like zone="graveyard" could be implemented for this?

**Comment by tooomm** [comment]
This is probably better discussed in Cockatrice directly as the client would need change in order to understand such tags.

--> Moved to Cockatrice/Cockatrice main repo

**Comment by Daenyth** [comment]
> We need a way to create a token that has "Destroy token when it leaves the table" disabled. I would suggest a new tag for cards like "movable".

I'm 99% sure this is already possible when you manually create a token. Is all that's missing an entry in tokens.xml to set the flag?

**Comment by tooomm** [comment]
>I'm 99% sure this is already possible when you manually create a token.

That is correct.
![token](https://user-images.githubusercontent.com/9874850/128529618-9a06cb15-6166-41e5-ba79-861b0c036eb8.png)

**Comment by mainman879** [comment]
> 
> 
> > We need a way to create a token that has "Destroy token when it leaves the table" disabled. I would suggest a new tag for cards like "movable".
> 
> I'm 99% sure this is already possible when you manually create a token. Is all that's missing an entry in tokens.xml to set the flag?

Yes as far as I know, there is no way to have the tokens.xml do this automatically when creating a token. I know the feature exists, I just don't think there is any way to have it be done automatically yet.

**Comment by ebbit1q** [comment]
duplicate of https://github.com/Cockatrice/Cockatrice/issues/4376

**Comment by ebbit1q** [comment]
creating permanent cards too often would be problematic as there is no way to properly remove them again, card permanency is handled server side

**Comment by mainman879** [comment]
> creating permanent cards too often would be problematic as there is no way to properly remove them again, card permanency is handled server side

If the permanency is done through token relations and the tokens.xml there will basically never be a time where this is done in error. These cards will be played by extremely few people and I doubt they'll make that many errors when making tokens from the context menu.

The "duplicate" issue also only attacks about allowing tokens on the stack. Not about allowing the tokens.xml to generate tokens to any zone.

**Comment by Tvpattack** [comment]
I'd like to bump this, as it's an issue I've come across quite regularly. I've had playgroups of 6+ people talk to me about wanting it to be automatic instead of needing to search it out or have the cards be predefined

**Comment by tooomm** [comment]
>We need a way to create a token that has "Destroy token when it leaves the table" disabled. I would suggest a new tag for cards like "movable".

That's basically #4161
